scylla/src/frame/response/result.rs

```diff
@@ -355,14 +384,35 @@ fn deser_cql_value(typ: &ColumnType, buf: &mut &[u8]) -> StdResult<CQLValue, Par
                 )));
             }
         }),
-        Set(typ) => {
+        List(type_name) => {
+            let len: usize = types::read_int(buf)?.try_into()?;
+            let mut res = Vec::with_capacity(len);
+            for _ in 0..len {
+                // TODO: is `null` allowed as list element? Should we use read_bytes_opt?
```

Comment:
According to @psarna's [earlier review comment](https://github.com/scylladb/scylla-rust-driver/pull/134#discussion_r546773285), the comment in the code is unnecessary. If you wish, you can also change the comment so that it mentions that lists cannot have NULL values as elements, and because of that we use `read_bytes` here.

Reply:
Done